src/ConnectionAutomaton.cpp

```diff
@@ -320,13 +320,13 @@ void ConnectionAutomaton::onConnectionFrame(
   switch (type) {
     case FrameType::KEEPALIVE: {
       Frame_KEEPALIVE frame;
-      if (!deserializeFrameOrError(frame, std::move(payload))) {
+      if (!deserializeFrameOrError(isResumable_, frame, std::move(payload))) {
```

Comment:
Reading the code is seems like isResumable_ is indeed fixed, there is even a TODO to add a check that client and server agrees:
> // TODO(tmont): check for ENABLE_RESUME and make sure isResumable_ is true

Spec also mentions this:
> NOTE: A server that receives a SETUP frame that has resume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR.

Seems like the behavior in this PR is correct, we should however support reconnecting without resumption (this isn't implemented).